source/features/toggle-files-button.tsx

```diff
@@ -27,6 +27,16 @@ const noticeClass = 'rgh-files-hidden-notice';
 const noticeStyle = {paddingRight: '16px'};
 
 function addButton(commitsLink: HTMLElement): void {
+	if (pageDetect.isSingleFile()) {
```

Comment:
Why here and not include?

Reply:
Is better to put it in `exclude`?

Reply:
No it cannot be moved because we still don't have a "page changed" listener, so the observer is only instantiated once (#6554):

1. open https://github.com/refined-github/refined-github/blob/main/test/setup-file.js
2. visit folder
3. no repotree features

Anyway, this feature makes no sense outside the homepage:

![form](https://github.com/user-attachments/assets/2e2b2d51-7314-44e0-8cea-065afc6bca1a)

And arguably it makes little sense elsewhere.

Merging this PR as is for now but I might drop the feature

Reply:
Opened
-  https://github.com/refined-github/refined-github/pull/7741